CMakeLists.txt

```diff
@@ -57,6 +57,14 @@ if(HERMES_APPLE_TARGET_PLATFORM MATCHES "catalyst")
   set(THREADS_PREFER_PTHREAD_FLAG ON)
 endif()
 
+if(HERMES_APPLE_TARGET_PLATFORM MATCHES "appletvos")
+  add_definitions(-DTARGET_OS_TV=1)
```

Comment:
Why do we need to define this ourselves, that is, why doesn't it defined automatically as it is for iOS?

Reply:
I don't know the answer to that question..... I just found that it was needed.

Reply:
I'll back out the node-hermes changes and see if CI succeeds and has the tvOS artifacts.

Reply:
I think it would be great to understand what's going on with this. Based on some cursory searches, it looks like the tvOS define should be in `TargetConditionals.h` just like the iOS one. This may suggest something is wrong with how the build is set up, or warrant a comment to avoid confusion in the future.

Reply:
I will have a look at `TargetConditionals.h`

Reply:
I am not able to determine why the build is not picking up the tvOS version of `TargetConditionals.h`. I will add a comment here and hopefully we can figure that out in the future.

Reply:
I apologise for the back and forth on this. But we can't merge changes that we don't really understand. In this instance, it may suggest a subtle issue in the build setup that could break in the future, or add overhead when managing the integration of our Apple builds with upstream RN.

Reply:
I totally understand. I'll close this for now, and once the React Native TV 0.71 release is out, will come back to this and see what is going on with TargetConditionals. My guess is that somewhere in the Hermes build, the Apple CMake configuration is picking up iPhone includes instead of Apple TV includes.